src/code/z_vr_box.c

```diff
@@ -1,24 +1,24 @@
 #include "global.h"
 
-u32 D_801C5E30[] = { 0, 0x2000, 0x4000, 0x6000, 0x8000, 0xC000 };
+u32 Skybox_TextureOffsets[] = { 0, 0x2000, 0x4000, 0x6000, 0x8000, 0xC000 };
 
-u16 D_801C5E48[] = {
+u16 Skybox_VertexOrder[] = {
     0, 2, 10, 12, 2, 4, 12, 14, 10, 12, 20, 22, 12, 14, 22, 24, 1, 3, 5, 6, 7, 8, 9, 11, 13, 15, 16, 17, 18, 19, 21, 23,
 };
-s32 D_801C5E88[] = { 0, 62 << 5, 124 << 5, 186 << 5, 248 << 5 };
 
-s32 D_801C5E9C[] = { 0, 62 << 5, 124 << 5, 186 << 5, 248 << 5 };
+s32 Skybox_texUOffsets[] = { 0, 62 << 5, 124 << 5, 186 << 5, 248 << 5 };
 
-s32 D_801C5EB0[] = { 0, 62 << 5, 124 << 5, 62 << 5, 0 };
+s32 Skybox_texVOffsets[] = { 0, 62 << 5, 124 << 5, 186 << 5, 248 << 5 };
 
-s16 D_801C5EC4[] = {
+s32 Skybox_texVOffsetsAlt[] = { 0, 62 << 5, 124 << 5, 62 << 5, 0 };
+
+s16 Skybox_VertexMapping[] = {
```

Comment:
This names don't follow the style of the project.
I suggest you to take a look at the STYLE document https://github.com/zeldaret/mm/blob/main/docs/STYLE.md